ui/pages/swaps/smart-transaction-status/index.scss

```diff
@@ -0,0 +1,83 @@
+@keyframes shift {
+  to {
+    background-position: 100% 0;
+  }
+}
+
+.smart-transaction-status {
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+  flex: 1;
+  width: 100%;
+
+  &__loading-bar-container {
+    height: 3px;
+    background: var(--Grey-100);
+    display: flex;
+    margin-top: 12px;
+    margin-bottom: 28px;
+  }
+
+  &__loading-bar {
+    height: 3px;
+    background: var(--Blue-500);
+    -webkit-transition: width 0.5s linear;
+    -moz-transition: width 0.5s linear;
+    -o-transition: width 0.5s linear;
+    transition: width 0.5s linear;
```

Comment:
I don't think we should need the browser prefixes ; `transition` has been around for several years now.